File: beecrowd/1165.c
#include <stdio.h>

int main() {
    
    int testes, divisores, numero;

    scanf("%d", &testes);

    for (int i = 0; i < testes; i += 1) {
        divisores = 0;
        scanf("%d", &numero);
        for (int v = 1; v <= numero; v += 1) {
            if (numero % v == 0) {
                divisores += 1;
            }
        }
        if (divisores <= 2)
            printf("%d eh primo\n", numero);
        else
            printf("%d nao eh primo\n", numero);
        
    }
    
    return 0;
}

File: beecrowd/1165.py
teste = int(input())
for c in range(teste):
    divisores = 0
    numero = int(input())
    for v in range(1, numero + 1):
        if numero % v == 0:
            divisores += 1
    if divisores > 2:
        print(f'{numero} nao eh primo')
    else:
        print(f'{numero} eh primo')